Sources/Commands/PackageTools/ArchiveSource.swift

```diff
@@ -67,7 +67,7 @@ extension SwiftPackageTool {
     ) throws  {
         let gitRepositoryProvider = GitRepositoryProvider()
         if gitRepositoryProvider.repositoryExists(at: packageDirectory) &&
-            gitRepositoryProvider.isValidDirectory(packageDirectory){
+            (try? gitRepositoryProvider.isValidDirectory(packageDirectory)) == true {
```

Comment:
Existing tests rely on the error not being fatal here, so this keeps the behavior.